### PR TITLE
Updated GSoC 2022 student blog for project Phoenix

### PR DESCRIPTION
**Pull request by DamianArado**
cc: @9inpachi, @EdwardMoyse 
for: https://github.com/hsf/phoenix

### CODE REVIEW

**Comment by 9inpachi** [comment]
Looks good. 👍

File: _gsocblogs/2022/blog_Phoenix_MohammadHumayunKhan.md [hunk 1]
```diff
@@ -27,20 +27,38 @@ Most of my time hence was spent doing the aforementioned things. I read about Ty
...
 
-Currently, I am doing the migration of the unit tests inside [phoenix-ng](https://github.com/HSF/phoenix/blob/master/packages/phoenix-ng/README.md) package from Jasmine and Karma setup to use Jest as well. This will eventually help us to use a single test runner for all our unit tests and it will also be easily maintainable by the developers. I hope that using Jest, we can find solutions to a lot of problems that we faced earlier with unit tests in Phoenix.
+#### Relevant Links
 
-So, this is a short report of all the work that has been done in this project till the midterm evaluation. If you want to read in detail about the same, you can visit [my repo on GitHub](https://github.com/DamianArado/GSoC-2022-Phoenix/blob/main/PROGRESS.md).
+- Project Work Summary: https://github.com/DamianArado/GSoC-2022-Phoenix/blob/main/SUMMARY.md
+- Project Roadmap: https://github.com/DamianArado/GSoC-2022-Phoenix/blob/main/ROADMAP.md
+- Project Progress: https://github.com/DamianArado/GSoC-2022-Phoenix/blob/main/PROGRESS.md
```
Comment (agheata) [inline _gsocblogs/2022/blog_Phoenix_MohammadHumayunKhan.md:62]: Please use `[Text](link)` format to make these links clickable

**Review by agheata — CHANGES_REQUESTED** [review]
@DamianArado after fixing the links, please ask your mentor to approve

File: _gsocblogs/2022/blog_Phoenix_MohammadHumayunKhan.md (hunk 1 above)
Comment (DamianArado) [reply _gsocblogs/2022/blog_Phoenix_MohammadHumayunKhan.md]: Done. ✅

**Review by agheata — APPROVED** [review]

**Pull request merged**